src/service/bind-to.js

```diff
@@ -33,7 +39,10 @@ angular.module('ngTasty.service.bindTo', [])
           scope[isolateScopeName] = parentValue;
         } else {
           // if the parent can be assigned then do so
-          parentSet(scope.$parent, parentValue = scope[isolateScopeName]);
+            if(isFunction(parentSet)){
+                parentSet(scope.$parent, parentValue = scope[isolateScopeName]);
```

Comment:
Looks your `parentGet.assign` is undefined, can you show me your case, so we can recreate it and write a test. ThankYou